incubator/storm/templates/_helpers.tpl

```diff
@@ -0,0 +1,75 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
This is not based on the latest `_helpers.tpl` . Please update.

Reply:
@unguiculus - Running version 2.9.1 of helm, when I exec the command `helm create` in the terminal, that line is still included in the `_helpers.tpl`.  

When evaluating other charts in both stable and incubator folders, I still that line included in all of them.  

* [stable/Artifactory](https://github.com/kubernetes/charts/blob/master/stable/artifactory/templates/_helpers.tpl)
* [stable/datadog](https://github.com/kubernetes/charts/blob/master/stable/datadog/templates/_helpers.tpl)

Could you please advise as to the update that you are looking to see?

Reply:
This line is not a problem. Please update the fullname templates.